From: =?utf-8?q?Thomas_M=C3=BChlbacher?= <tmuehlbacher@posteo.net>
Date: Thu, 28 Jan 2021 16:03:10 +0100
Subject: tests/monitor-config: Improve debugging output

Make it easier to find out what went wrong with `migrated_data` by
having it included in the debug logs.

Bug: <https://gitlab.gnome.org/GNOME/mutter/-/issues/1011>
(cherry picked from commit 180e62519b4697ef263a28aa6a3db746627c0a4d)
Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1694>
Origin: upstream, 3.38.4, commit:015acbbc30b7d00e5c36a1d21ea9dc40c1e47e0a
---
 src/tests/monitor-config-migration-unit-tests.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tests/monitor-config-migration-unit-tests.c b/src/tests/monitor-config-migration-unit-tests.c
index 461035d..bb2ac62 100644
--- a/src/tests/monitor-config-migration-unit-tests.c
+++ b/src/tests/monitor-config-migration-unit-tests.c
@@ -75,7 +75,7 @@ test_migration (const char *old_config,
   g_assert_nonnull (expected_data);
   g_assert_nonnull (migrated_data);
 
-  g_assert (strcmp (expected_data, migrated_data) == 0);
+  g_assert_cmpstr (expected_data, ==, migrated_data);
 
   migrated_file = g_file_new_for_path (migrated_path);
   if (!g_file_delete (migrated_file, NULL, &error))
